### PR TITLE
Add Remaining Documentation for System.Drawing for .NET 9

### PR DESCRIPTION
**Pull request by lonitra**
## Summary

Related: https://github.com/dotnet/winforms/issues/11858

### CODE REVIEW

**Review by gewarren — APPROVED** [review]

File: xml/System.Drawing.Imaging.Effects/BlurEffect.xml [hunk 1]
```diff
@@ -36,9 +36,11 @@
       </Parameters>
       <Docs>
         <param name="radius">
-          <para>
             <see langword="true" /> to expand the bitmap by an amount equal to the blur radius so it can have soft edges. <see langword="false" /> to keep the bitmap the same size and clip the soft edges.
```
[the remark this answers is not in the document]
Comment (lonitra) [reply xml/System.Drawing.Imaging.Effects/BlurEffect.xml:39]: Nice catch, thank you!

File: xml/System.Drawing.Imaging.Effects/Effect.xml [hunk 2]
```diff
@@ -47,12 +47,12 @@
...
-      <MemberSignature Language="C#" Value="public virtual void Dispose (bool disposing);" />
-      <MemberSignature Language="ILAsm" Value=".method public hidebysig newslot virtual instance void Dispose(bool disposing) cil managed" />
+      <MemberSignature Language="C#" Value="protected virtual void Dispose (bool disposing);" />
+      <MemberSignature Language="ILAsm" Value=".method protected hidebysig newslot virtual instance void Dispose(bool disposing) cil managed" />
       <MemberSignature Language="DocId" Value="M:System.Drawing.Imaging.Effects.Effect.Dispose(System.Boolean)" />
-      <MemberSignature Language="VB.NET" Value="Public Overridable Sub Dispose (disposing As Boolean)" />
+      <MemberSignature Language="VB.NET" Value="protected Overridable Sub Dispose (disposing As Boolean)" />
       <MemberSignature Language="F#" Value="abstract member Dispose : bool -&gt; unit&#xA;override this.Dispose : bool -&gt; unit" Usage="effect.Dispose disposing" />
-      <MemberSignature Language="C++ CLI" Value="public:&#xA; virtual void Dispose(bool disposing);" />
+      <MemberSignature Language="C++ CLI" Value="protected:&#xA; virtual void Dispose(bool disposing);" />
```
[the remark this answers is not in the document]
Comment (lonitra) [reply xml/System.Drawing.Imaging.Effects/Effect.xml:55]: This has been fixed in our branch to be protected (https://github.com/dotnet/winforms/pull/12090). Is it ok to fix it here like this? Or should this be reverted, and the next CI update will fix this?

File: xml/System.Drawing.Imaging/ColorPalette.xml [hunk 3]
```diff
@@ -70,14 +70,6 @@
         <AssemblyName>System.Drawing</AssemblyName>
       </AssemblyInfo>
       <Parameters>
-        <Parameter Name="entries" Type="System.Drawing.Color[]" Index="0" FrameworkAlternate="dotnet-plat-ext-9.0">
```
[the remark this answers is not in the document]
Comment (lonitra) [reply xml/System.Drawing.Imaging/ColorPalette.xml:73]: ColorPalette does not have this parameter in its public constructor so I am not sure why this is being auto generated: https://github.com/dotnet/winforms/blob/release/9.0/src/System.Drawing.Common/src/System/Drawing/Imaging/ColorPalette.cs#L38

File: xml/System.Drawing.Imaging.Effects/BlurEffect.xml (hunk 1 above)
Comment (lonitra) [inline xml/System.Drawing.Imaging.Effects/BlurEffect.xml:39]:
```suggestion
            Real number that specifies the blur radius (the radius of the Gaussian convolution kernel) in pixels. The
            radius must be in the range 0 through 256. As the radius increases, the resulting bitmap becomes more blurry.
```

File: xml/System.Drawing.Imaging.Effects/Effect.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (gewarren) [reply xml/System.Drawing.Imaging.Effects/Effect.xml:55]: It's okay to change it here, but the CI will do whatever it wants to do when I run it later this week.

File: xml/System.Drawing.Imaging/ColorPalette.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (gewarren) [reply xml/System.Drawing.Imaging/ColorPalette.xml:73]: Ahh okay, that is specific to a "framework" that we deleted, so it's fine to delete it. It should have been cleaned up by the CI ideally.

File: xml/System.Drawing.Imaging.Effects/Effect.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (lonitra) [reply xml/System.Drawing.Imaging.Effects/Effect.xml:55]: Sounds good. I'll make sure to take a look at the CI update later this week when it is out. btw - I do not have permissions to merge, so if the change looks good could you merge?

**Pull request merged**